components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -1849,7 +1849,7 @@
         var value = input.value;
         var text = element.down('.value').firstChild.nodeValue;
         var ynpickerElt = YesNoPicker.generatePickerElement([
-              {type: 'na',                         selected: !isValueSelected(positiveName, value) &amp;&amp; !isValueSelected(negativeName, value)},
+              {type: 'na' , name: positiveName + '_na', selected: !isValueSelected(positiveName, value) &amp;&amp; !isValueSelected(negativeName, value)},
```

Comment:
That's not very good, since now extra useless data will be sent to the server whenever saving. Isn't there another way, without giving names to NA elements?